src/backend/distributed/shared_library_init.c

```diff
@@ -693,6 +693,17 @@ RegisterCitusConfigVariables(void)
 		GUC_NO_SHOW_ALL,
 		NoticeIfSubqueryPushdownEnabled, NULL, NULL);
 
+	DefineCustomRealVariable(
+		"citus.local_node_parallel_execution_factor",
+		gettext_noop("TODO:"),
```

Comment:
poke

Reply:
> I guess you are right, especially considering that we already have citus.max_shared_pool_size, it'd be preferable to be inline with that.
What do you think about `citus.local_shared_pool_size'? It might make the explanation easier?

I'm waiting an OK for you on the suggested name before updating it. Any comments?

Reply:
Alright, for now I pushed `citus.local_shared_pool_size` with a very similar semantics of `citus.max_shared_pool_size`. Below is a comment from the source:

```C
/*
 * Controlled via a GUC, never access directly, use GetLocalSharedPoolSize().
 *  "0" means adjust LocalSharedPoolSize automatically by using MaxConnections.
 * "-1" means do not apply connection throttling
 * Anything else means use that number
 */
```